go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/bitrise-io/appcenter
 go 1.12
```

Comment:
nit: How about upgrading this to 1.18? :D

Reply:
will update in the step where it is vendored, it will not make much difference here so leaving as is.